DFE-Digital/early-years-foundation-reform
# HFEYP-623 change from flexbox to grid so last line of grid aligns left

## Ticket and context

Ticket: [HFEYP-623](https://dfedigital.atlassian.net/browse/HFEYP-623)
## Tech review
Change from flexbox to grid. This means that last line of the grid will align left rather than centre. 
### Is there anything that the code reviewer should know?

### Code quality checks

## Product review

### How can someone see it in review app?
1. Click [this link](https://eyfs-cms-review-pr-545.london.cloudapps.digital/articles) or navigate to `/articles` in the review app (posted by a `github-actions` bot below)
2. Verify that the links to articles are displaying as required. Adjust the size of the screen to make sure it is responsive.
![image](https://user-images.githubusercontent.com/13471320/148938951-43324b1f-b62f-4df2-b4b6-d0a2c173e91f.png)
![image](https://user-images.githubusercontent.com/13471320/148939050-3c2d7fc4-19e4-410c-9535-35752020b658.png)
![image](https://user-images.githubusercontent.com/13471320/148939090-0c77dcbc-2376-4102-b9b1-b4da2949cb62.png)
